roles/ceph-osd/tasks/start_osds.yml

```diff
@@ -32,5 +34,6 @@
   service:
     name: ceph-osd@{{ item }}
     state: started
+    enabled: true
```

Comment:
You are right, I missed that they are activated in a separate file. The other daemons do have that line in them (enabled: true, at least mon and mgr that I checked) -- the installation would not work without it, nothing would start after reboot. It is only the osds that were treated specially because of the way they are usually activated.